server/quizzes/routes.js

```diff
@@ -14,3 +17,12 @@ quizRouter.get('/', async (req, res) => {
       res.status(500).json({ error: 'something went wrong retreving quizzes from db.' });
     });
 });
+
+quizRouter.post('/', authMiddleWare, async (req, res) => {
```

Comment:
will this end up blocking the quiz route completely without authentication? Maybe we need to set up tokens for anonymous users so that they can still access quiz routes but only through the front end? Maybe you already did that and I didn't notice.

Reply:
Right now, it's only for the `post` route to `/api/quizzes` - the others will still work as this middleware applies only to this specific route